stable/redmine/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redmine
-version: 0.4.3
+version: 0.5.3
```

Comment:
Yeah, I guess @tompizmor forgot to reset the patch version. Actually this is a breaking change with the env var changes right @tompizmor? In that case we should bump the major version so this should be `1.0.0`